lib/client/keystore_test.go

```diff
@@ -317,6 +318,33 @@ func TestCheckKeyFIPS(t *testing.T) {
 	require.True(t, trace.IsBadParameter(err))
 }
 
+func TestGetTrustedCertsPEM_nonCertificateBlocks(t *testing.T) {
+	s, cleanup := newTest(t)
+	defer cleanup()
+
+	// Make sure we behave correctly if someone writes a non-CERTIFICATE block to
+	// certs.pem. During regular use this shouldn't happen, but a bug was lurking
+	// around here.
+	proxy := "proxy.example.com"
+	proxyDir := filepath.Join(s.storeDir, "keys", proxy)
+	certsFile := filepath.Join(proxyDir, "certs.pem")
+	err := os.MkdirAll(proxyDir, 0700)
+	require.NoError(t, err)
```

Comment:
Instead of building the paths manually consider using the helpers from `api/utils/keypaths`:
```suggestion
	err := os.MkdirAll(keypaths.ProxyKeyDir(s.storeDir, proxy), 0700)
	require.NoError(t, err)
	certsFile := keypaths.TLSCAsPath(s.storeDir, proxy)
```

Reply:
Done - applied a slightly modified version, using `os.MkdirAll(filepath.Dir(certsFile), 0700)` instead.